commit dfab74d74daf7ff601ad88d66b1096c4b3cbd1c3
Author: Christoph Berg <myon@debian.org>
Date:   Tue Oct 12 21:55:18 2021 +0200

    ic7610: Re-add send_morse
    
    Commit 28b7543de660c432b22d0035b250d32fb91797e4 mistakenly removed
    send_morse from ic7610, add it back.

diff --git a/rigs/icom/ic7610.c b/rigs/icom/ic7610.c
index 1a764666..32940e9a 100644
--- a/rigs/icom/ic7610.c
+++ b/rigs/icom/ic7610.c
@@ -475,6 +475,7 @@ const struct rig_caps ic7610_caps =
     .get_split_vfo =  icom_get_split_vfo,
     .set_powerstat = icom_set_powerstat,
     .get_powerstat = icom_get_powerstat,
+    .send_morse = icom_send_morse,
     .stop_morse = icom_stop_morse,
     .wait_morse = rig_wait_morse
 };
